docs/key-binding.md

```diff
@@ -0,0 +1,38 @@
+# Key binding
+
+By default, Atuin will rebind both Ctrl-r and the up arrow. If you do not want
```

Comment:
Hey, did you know that GitHub allows you to use the HTML tag `kbd` to make text look like a <kbd>keyboard button</kbd>?
```suggestion
By default, Atuin will rebind both <kbd>Ctrl-r</kbd> and the up arrow. If you do not want
```

Reply:
I did not! That looks much better, thank you